helpers/corpus.py

```diff
@@ -0,0 +1,89 @@
+import json
+import random
+from shared import config
+
+
+class Corpus:
```

Comment:
This is really good and could potentially in the future be bumped down the stack to hagadias.
It could use an explanatory class-level comment.